### PR TITLE
[Format] Apply PEP8 to pcapdnet.py

### PR DESCRIPTION
**Pull request by gpotter2**
Current pcapdnet.py file is un-readable...
This PR changes nothing but `pcapdnet.py` style + format:
- unify indenting: 4 spaces
- Auto PEP8
- Remove tripple duplicated `PcapTimeoutElapsed`
- use scapy.consts.WINDOWS instead of loading it again
- separate more readibily the different OS

### CODE REVIEW

**Comment by gpotter2** [comment]
@guedou It seems that this will also fix the current pcapdnet bug on the `linux SUDO PCAPDNET` tests (when the iface does not exist, it crashes unexpectedly)

File: scapy/arch/pcapdnet.py [hunk 1]
```diff
@@ -548,19 +631,23 @@ def get_if_raw_addr(ifname):
             i = dnet.intf()
             try:
                 return i.get(ifname)["addr"].data
-            except OSError:
+            except (OSError, KeyError):
```
Comment (gpotter2) [inline scapy/arch/pcapdnet.py]: I may include this fix in a different PR if really needed

Comment (guedou) [reply scapy/arch/pcapdnet.py]: Can you move it to another PR ? Thanks.

**Comment by p-l-** [comment]
@gpotter2 we do not want such large "PEP-8" or style commits, mainly for code history. Maybe one day we will have one big scapy-wide commit to handle that...

**Comment by gpotter2** [comment]
@p-l- I useally agree on this rule, but this file especially is incredibly hard to read :/
@guedou Created a new PR

**Pull request closed without merging**